package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@edx/edx-bootstrap": "0.4.3",
     "@edx/paragon": "1.7.1",
-    "@edx/studio-frontend": "0.7.0",
+    "@edx/studio-frontend": "0.8.3",
```

Comment:
@thallada did this need a corresponding `package-lock.json` change?

It looks [like `@edx/studio-frontend` is pinned to `v.0.7.0` in `master`.](https://github.com/edx/edx-platform/blob/master/package-lock.json#L64-L113)

![image](https://user-images.githubusercontent.com/8136030/35101167-89041bea-fc2c-11e7-92a1-73accb66d926.png)

Reply:
Created a PR with updated lockfile: #17233

Reply:
Thanks Jae. Can't believe I forgot that again...